turbinia/turbiniactl.py

```diff
@@ -883,6 +896,9 @@ def process_evidence(
   if args.command == 'rawdisk':
     evidence_ = evidence.RawDisk(
         name=name, source_path=os.path.abspath(source_path), source=source)
+  elif args.command == 'expertwitnessformat':
```

Comment:
FYI: @jleaniz We'll want to add this to the API server evidence definitions too.

Reply:
thx for pointing this out. Will do